Docs/Configuration.tex

```diff
@@ -265,35 +265,35 @@ \subsection{Configuration Terms}\label{configuration-terms}
   \texttt{invalid\ value} if missing.
 \item
   \texttt{fatal\ behaviour} --- behaviour leading to boot termination.
-  Implementation must stop the boot process from going any further until
-  next host system boot. It is allowed but not required to perform cold
-  reboot or show any warning message.
+  Implementations must prevent the boot process from proceeding until the
```

Comment:
Perhaps better, 'continuing' instead of 'proceeding'?

Reply:
Concur